tests/codebuild/update_tests.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+source common.sh
+source inject_tests.sh
+
+# Updates the spec and re-applies to ensure updates work as expected.
+# Parameter:
+#    $1: Namespace of the CRD
+function run_update_integration_tests
```

Comment:
For the other files we split integration tests up into "canary tests" and "integration tests". I would think updating should be a canary test, at least for these basic cases?